src/meta/src/barrier/progress.rs

```diff
@@ -328,6 +357,7 @@ impl CreateMviewProgressTracker {
             progress_map,
             actor_map,
             pending_finished_jobs: Vec::new(),
+            pending_backfill_nodes,
```

Comment:
Seems more straight forward to avoid initially pausing all backfill nodes when recovery and just initialize with an empty `pending_backfill_nodes`.

Reply:
Fixed by https://github.com/risingwavelabs/risingwave/pull/20967/commits/305fab707d07e84e9e9f9ce5b51a90b528cf6a14

and https://github.com/risingwavelabs/risingwave/pull/20967/commits/a8f13b2806484eeffb6261855f50a3306e5e37ca